<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>report_form</class>
 <widget class="QWidget" name="report_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1018</width>
    <height>852</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Окно формирования отчёта</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="2">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Виды ЧС (по классу)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Класс ЧС</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Источник</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLineEdit" name="predicted">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QComboBox" name="emergency_type">
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Ответ эксперта</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Ответ системы о ЧС</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QComboBox" name="emergency_class">
     <item>
      <property name="text">
       <string>Техногенная</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Антропогенная</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Природная</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Экологическая</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Социальная</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Смешанная</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="source">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Предпринятые действия для предотвращения последствий ЧС или их минимизации</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QTextEdit" name="operator_actions"/>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="btn_create_filled_report">
        <property name="text">
         <string>Создать отчёт</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Дата получения</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLineEdit" name="actual"/>
   </item>
   <item row="5" column="0" colspan="4">
    <widget class="QWidget" name="just_widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#widget: border: 1px solid black;
</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Воспользуйтесь полями ниже, чтобы добавить описание ЧС и ваших действий для её предотвращение. Вы можете сохранить отчёт с незаполненными полями, чтобы заполнить их в вашем текстовом редакторе самостоятельно.</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>ID сообщения</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="sender_message">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="id_message">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Последствия ЧС и её масштаб</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QTextEdit" name="impact_and_scale"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Краткое описание ЧС, причины её возникновения</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QTextEdit" name="emergency_description"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Отправитель</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="receive_date"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
